src/org/labkey/test/pages/admin/ImportFolderPage.java

```diff
@@ -116,5 +118,6 @@ protected ElementCache newElementCache()
 
         WebElement importFolderButton = Locator.lkButton("Import Folder").findWhenNeeded(this);
         WebElement usePipelineButton = Locator.lkButton("Use Pipeline").findWhenNeeded(this);
+        WebElement manageFileWatcherTriggersButton = Locator.lkButton("Manage File Watcher Triggers").findWhenNeeded(this);
```

Comment:
just as an FYI, you can chain a timeout setter on finder functions- .findWhenNeeded(context).withTimeout(msec).
without the timeout, it resolves the element immediately when called (and will fail if it is not in the page at that time).  With it, a timer is started when we start to look.

it's irrelevant if that element is always going to be there by the time code gets to it, but it's a nice thing to know about :-)